document/document.go

```diff
@@ -97,6 +94,23 @@ func New() *Document {
 	return d
 }
 
+func (d *Document) GetOrCreateCustomProperties() common.CustomProperties {
```

Comment:
https://github.com/unidoc/unioffice/wiki/UniOffice-Developer-Guide#10-doc-comments

Reply:
Initially new document don't contain custom properties and we don't know if one does. GetOrCreateCustomProperties() is a shortcut of something like

cp := doc.CustomProperties
if cp.X() == nil {
  cp = common.NewCustomProperties
  doc.CustomProperties = cp
}

so it should be used when we definitely want document to have custom properties and we don't know if it is already does

Reply:
Initially new document don't contain custom properties and we don't know if one does. GetOrCreateCustomProperties() is a shortcut of something like

cp := doc.CustomProperties
if cp.X() == nil {
cp = common.NewCustomProperties
doc.CustomProperties = cp
}

so it should be used when we definitely want document to have custom properties and we don't know if it is already does. E.g.

cp := doc.GetOrCreateCustomProperties()

Reply:
added a comment to code according to the gudelines

Reply:
done